packages/core/src/resolver.ts

```diff
@@ -275,7 +275,7 @@ interface ResolverAuthorize {
   (...args: Parameters<SessionContextBase["$authorize"]>): <T, C>(
     input: T,
     ctx: C,
-  ) => ResultWithContext<T, AuthenticatedMiddlewareCtx>
+  ) => ResultWithContext<T, AuthenticatedMiddlewareCtx | Ctx>
```

Comment:
Thanks Cam, but this breaks the "typeguard" feature of authorize(). This return type must be AuthenticatedMiddlewareCtx.

Without knowing the code and error you're having, I'm not sure what the correct solution would be.